### PR TITLE
Create and style form and footer sections

### PR DESCRIPTION
**Pull request by Javivy**
In this milestone I: 
  - Create the structure of the form with HTML
  - Style the form with CSS
  - Mark the inputs as required 
  - Link the form with Formspree
  - Make use of flexbox to position the form correctly and in a responsive way with best practices
  - Name and make commits professionally

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -198,6 +198,53 @@ <h3>Skills</h3>
...
+        </h2>
+        <ul class="form-ul">
+          <li class="form-li">
+            <input autocomplete="off" maxlength="30" type="text" name="name" id="name" placeholder="Full Name" required>
+          </li>
+          <li class="form-li">
+            <input autocomplete="off" type="email" name="email-address" id="email-address" placeholder="Email address" required>
+          </li>
+          <li class="form-li">
+            <textarea autocomplete="off" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here"></textarea>
```
Comment (frank1738) [inline index.html:214]: - [ ] Please add an attribute `required` to your message input to ensure your form is not submitted with an empty message input field

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Javivy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Great Design 👍
✅ Nice PR title and description 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -198,6 +198,53 @@ <h3>Skills</h3>
...
+          <li class="form-li">
+            <input autocomplete="off" type="email" name="email-address" id="email-address" placeholder="Email address" required>
+          </li>
+          <li class="form-li">
+            <textarea autocomplete="off" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+          </li>
+          <li class="form-li">
+            <button type="submit" class="green-btn form-btn">
+              Get in touch
+            </button>
```
Comment (1GORDON) [inline index.html:219]: Great design @Javivy 🤝

- [ ] [Optional] Kindly implement all button interactions of `enable`, and `hover` as specified in the project requirements.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Javivy ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**